### PR TITLE
PLFM-5561

### PR DESCRIPTION
**Pull request by nickgros**

### CODE REVIEW

File: services/repository/src/main/java/org/sagebionetworks/repo/web/controller/EvaluationController.java [hunk 1]
```diff
@@ -446,10 +446,18 @@ Submission createSubmission(
 		return serviceProvider.getEvaluationService().createSubmission(
 				userId, submission, entityEtag, submissionEligibilityHash, request, challengeEndpoint, notificationUnsubscribeEndpoint);
 	}
-	
+
+	@Deprecated
+	@ResponseStatus(HttpStatus.CREATED)
```
Comment (brucehoff) [inline services/repository/src/main/java/org/sagebionetworks/repo/web/controller/EvaluationController.java]: GONE

File: services/repository/src/main/java/org/sagebionetworks/repo/web/controller/MessageController.java [hunk 2]
```diff
@@ -250,18 +250,25 @@ public void updateMessageStatus(
 			@RequestBody MessageStatus status) throws NotFoundException {
 		serviceProvider.getMessageService().updateMessageStatus(userId, status);
 	}
-	
+
+	@Deprecated
+	@ResponseStatus(HttpStatus.OK)
```
Comment (brucehoff) [inline services/repository/src/main/java/org/sagebionetworks/repo/web/controller/MessageController.java]: GONE

**Comment by nickgros** [comment]
Deprecated endpoints:

* 'DELETE /certifiedUserTestResponse/#'
* 'GET /certifiedUserTestResponse'
* 'GET /user/#/certifiedUserPassingRecords'
* 'DELETE /message/#'
* 'PUT /user/#/certificationStatus'
* 'POST /evaluation/submission/#/contributor'

**Review by brucehoff — APPROVED** [review]
Looks great.

**Pull request merged**